tests/abort1.rs

```diff
@@ -127,7 +127,16 @@ async fn test_stop_taker_after_setup() {
         .collect::<Vec<_>>();
 
     // Start swap
-    thread::sleep(Duration::from_secs(360)); // Take a delay because Makers take time to fully setup.
+
+    // Makers take time to fully setup.
+    makers.iter().for_each(|maker| {
+        while !*maker.is_setup_complete.read().unwrap() {
+            // Introduce a delay of 10 units to prevent write lock starvation.
+            thread::sleep(Duration::from_secs(10));
```

Comment:
One info log here like "waiting for maker setup completion" would be helpful.

Reply:
Cool will resolve this in next commit